components/setWorkoutDetails/SetWorkoutDetailsCard.jsx

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable radix, max-len */
+import React from 'react';
+import { View } from 'react-native';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import { AntDesign } from '@expo/vector-icons';
+
+import EllipsisPopup from '../utils/EllipsisPopup';
+
+const NameText = styled.Text`
+   color: #EFEFEF;
+   font-size: 24px;
+   font-family: 'Montserrat_500Medium';
+`;
+
+const StyledEllipsesPopup = styled(EllipsisPopup)`
+   position: absolute;
+   right: 10px;
+   top: 18px;
+   height: 25px;
+   width: 25px;
+`;
+
+const StyledIcon = styled(AntDesign)`
+   padding: 0px 10px;
+`;
+
+const ContentWrapper = styled.View`
+  flexDirection: row;
+`;
+const StyledInput = styled.TextInput`
+  width: 70px;
+  height: 25px;
+  background-color: #FFFFFF;
+  border-radius: 5px;
+  text-align: center;
+  font-family: 'Roboto_400Regular';
+  font-size: 16px;
+`;
+
+const StyledText = styled.Text`
+  font-family: 'Roboto_400Regular';
+  color: #FFF;
+  font-size: 18px;
+  height: 25px;
+  text-align: center;
+  padding: 2px 15px; 0px 0px;
+`;
+
+const StyledView = styled.View`
+  width: 90%;
+  height: 120px;
+  border-radius: 20px;
+  padding: 15px;
+  margin: 10px 5%;
+  box-shadow: 3px 5px 2px #00000050;
+`;
+
+const SetWorkoutDetailsCard = (props) => {
+  const {
+    color, name, displayEllipses,
+  } = props;
+  const [sets, setSets] = React.useState('');
+  const [reps, setReps] = React.useState('');
+  const [seconds, setSeconds] = React.useState('');
+  const [isReps, setIsReps] = React.useState(true);
```

Comment:
I think you need to move your state a little higher and into the CreateWorkout component so you can access this information when the finish button is clicked and send it to the database